src/trackinfoobject.cpp

```diff
@@ -783,13 +783,14 @@ bool TrackInfoObject::isDirty() {
 
 int TrackInfoObject::getRating() const {
     QMutexLocker lock(&m_qMutex);
-    return m_iRating;
+    return m_metadata.getRating();
```

Comment:
Hmm, daschuer, can you make the necessary changes? Not sure how to go about them.
And are you suggesting a 'sync rating to stars' option in interface or something?